src/c++/perf_analyzer/command_line_parser.h

```diff
@@ -127,6 +127,8 @@ struct PerfAnalyzerParameters {
         using_concurrency_range || using_old_options ||
         !(using_request_rate_range || using_custom_intervals));
   }
+
+  double overhead_pct_threshold{10.0};
```

Comment:
Please put a comment for what this is for.  10% seems very low for this first pass (since we havne't fixed it yet, and havne't added control to override). have you run it at all to see if it gets hit on normal runs? I'm wondering if we should start out with something quite high like 50%